setup.py

```diff
@@ -25,25 +25,24 @@ def load_about():
 
 
 setup(
-    name="drydock",
+    name="drydock-backups",
     version=ABOUT["__version__"],
-    url="https://github.com/edunext/drydock",
+    url="https://github.com/edunext/drydock-backups",
     project_urls={
-        "Code": "https://github.com/edunext/drydock",
-        "Issue tracker": "https://github.com/edunext/drydock/issues",
+        "Code": "https://github.com/edunext/drydock-backups",
+        "Issue tracker": "https://github.com/edunext/drydock-backups/issues",
     },
     license="AGPLv3",
     author="eduNEXT",
-    description="A Tutor plugin to manage our opinionated Open edX operations",
+    description="Drydock backups plugin for Tutor",
     long_description=load_readme(),
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    python_requires=">=3.7",
-    install_requires=["tutor", "packaging"],
+    python_requires=">=3.8",
+    install_requires=["tutor>=16.0.0,<17.0.0"],
```

Comment:
I'm not pretty sure if we should use the same version as Tutor for this plugin. I think that a single version could be compatible with multiple Tutor versions. what do you think? cc @MoisesGSalas